Project_One/app/src/main/java/com/justinwells/project_one/RecyclerViewAdapter.java

```diff
@@ -0,0 +1,64 @@
+package com.justinwells.project_one;
+
+import android.graphics.Color;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import java.util.List;
+
+/**
+ * Created by justinwells on 10/21/16.
+ */
+
+public class RecyclerViewAdapter extends RecyclerView.Adapter<CustomViewHolder> {
+
+    List<ToDoItem> choreList;
+    private int selectedPos = 0;
+
+
+    public RecyclerViewAdapter(List<ToDoItem> choreList) {
+        this.choreList = choreList;
+    }
+
+    @Override
+    public CustomViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View parentView = LayoutInflater.from(parent.getContext()).inflate(R.layout.list_object,
+                parent,false);
+        CustomViewHolder viewHolder = new CustomViewHolder(parentView);
+        return  viewHolder;
+    }
+
+    @Override
+    public void onBindViewHolder(final CustomViewHolder holder, final int position) {
+        holder.chore.setText(choreList.get(position).getToDo());
```

Comment:
It's totally fine to use `position` here, because this line of code runs right away when `onBindViewHolder()` is called